platform/ui/src/components/customForm/HotkeyField.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { utils } from '@ohif/core';
+
+const { hotkeys } = utils;
```

Comment:
Maybe we make `hotkeys` a top level export so we don't need to be weird here.

Reply:
We can have as top level, but does make sense to have as top level if its just a util object?

Reply:
I'm not sure. I think exposing it as top level makes it easier to code split individual utils at a later date? Open to suggestion

Reply:
Hotkeys are now exposed as root level. :)